### PR TITLE
Devel python3 and logging

### PR DESCRIPTION
**Pull request by kxyne**
This code fixes some minor boto python 3 compatibility changes, makes the prints into logging calls and changes the environment and config variables to refer to new, non clashing names.

config.json contains:
"rds_uri": "postgres://cybergreen:$RDS_PASSWORD@cg-stats-aggtest.crovisjepxcd.eu-west-1.rds.amazonaws.com:5432/$CYBERGREEN_BUILD_ENV",
    "redshift_uri": "postgres://cybergreen:$REDSHIFT_PASSWORD@cg-analysis-aggtest2.cqxchced59ta.eu-west-1.redshift.amazonaws.com:5439/$CYBERGREEN_BUILD_ENV",
    "role_arn": "arn:aws:iam::635396214416:role/RedshiftCopyUnload",
    "source_path": "$CYBERGREEN_SOURCE_ROOT",
    "dest_path": "$CYBERGREEN_DEST_ROOT",
    "agg_path": "$CYBERGREEN_AGG_ROOT",
    "access_key":"$AWS_ACCESS_KEY_ID",
    "secret_key":"$AWS_SECRET_ACCESS_KEY",

environment contains:
export CYBERGREEN_AGG_ROOT="s3://bits.cybergreen.net/test/stats/latest"
export RDS_PASSWORD='rdspass'
export REDSHIFT_PASSWORD='redshiftpass'
export CYBERGREEN_BUILD_ENV=test

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -310,12 +314,15 @@ def load_ref_data_rds(self):
...
             # So have to download localy and sasve (takes ~5 seconds)
             if 'asn' not in url:
-                push_datapackage(descriptor=url,backend='sql',engine=conn)
+                push_datapackage(descriptor=url, backend='sql', engine=conn)
             else:
                 dp = datapackage.DataPackage(url)
                 # local path will be returned if not found remote one (fot tests)
                 url = dp.resources[0].remote_data_path or dp.resources[0].local_data_path
-                urllib.urlretrieve(url, join(self.tmpdir, 'asn.csv'))
+                r = requests.get(url)
```
Comment (zelima) [inline main.py]: can we keep `urllib.urlretrieve` ? as for tests we load datapackage.json from `/test/fixtures/...`

This is the reason travis is failing

**Comment by zelima** [comment]
Also can we do not delete config.json, but keep and just modify variable names there

**Comment by kxyne** [comment]
restored config, urllib in python 3 is very different so we can't use it in the same way anyway, it's more like a thin layer around requests (urllib3).

**Comment by kxyne** [comment]
I've made some fixes, I think I did this wrong but I'll do another pull req.

**Comment by aaronkaplan** [comment]
That's exactly why the fork & make a PR & review by another pair of eyes - model on github makes so much sense.

**Pull request closed without merging**